src/navigation/RequestsNavigation.tsx

```diff
@@ -16,7 +16,7 @@ export const RequestsNavigation = () => (
     <RequestsStack.Screen
       name="SEE_REQUEST"
       component={SeeRequest}
-      options={{ ...TransitionPresets.ModalSlideFromBottomIOS }}
+      options={{ ...TransitionPresets.SlideFromRightIOS }}
```

Comment:
please delete redundant spread and braces, and apply GestureRecognizer to handle gestures in SeeRequest screen

Reply:
I'm waiting for a feedback from design team as I'm not sure do they want solution like this.